ci/terraform/shared/site.tf

```diff
@@ -59,7 +64,12 @@ provider "aws" {
 
   default_tags {
```

Comment:
Is this duplicated with above?  What is the backend that sits behind this and is it shared with Orch?

Reply:
yeah, this is so we can deploy stuff in `us-east-1`: we need a separate aws provider instance to achieve that.

I've kept the tags the same for simplicity, it may be that these are separate IRL, but unsure / this feels a bit too 'in the weeds' for this first-brush attempt of adding these tags. Happy to update if y'all want though.